aggregator.go

```diff
@@ -8,32 +8,33 @@ type (
 	// Aggregator can calculate some value across all netmap
 	// such as median, minimum or maximum.
 	Aggregator interface {
-		Add(Node)
+		Add(float64)
 		Compute() float64
+		Clear()
 	}
 
 	// Normalizer normalizes weight.
 	Normalizer interface {
 		Normalize(w float64) float64
 	}
 
-	meanCapSumAgg struct {
-		sum   uint64
+	meanSumAgg struct {
```

Comment:
What do you think about making these aggregators and normalizers public?

Reply:
I think it is unnecessary, we can just export constructors.